testcases/OpTestNVRAM.py

```diff
@@ -104,49 +86,15 @@ def test_nvram_configuration(self):
         self.cv_HOST.host_run_command("nvram --ascii lnx,oops-log")
         self.cv_HOST.host_run_command("nvram --ascii wwwwwwwwwwww")
         try:
-            self.test_nvram_update_part_config_in_host("common")
-            self.test_nvram_update_part_config_in_host("ibm,skiboot")
-            self.test_nvram_update_part_config_in_host("lnx,oops-log")
-            self.test_nvram_update_part_config_in_host("wwwwwwwwwwww")
+            self.nvram_update_part_config_in_host("common")
+            self.nvram_update_part_config_in_host("ibm,skiboot")
+            # The following 2 are disabled due to possible nvram binary
+            # bug.
+#            self.nvram_update_part_config_in_host("lnx,oops-log")
+#            self.nvram_update_part_config_in_host("wwwwwwwwwwww")
         except OpTestError:
```

Comment:
Ahh yes, in fact, I have a better way to fix this - have a custom Exception that we can catch. That way we get a nice error message, and we can do try/except/else to do negative tests (such as partition name too long).